Userland/Services/SpiceAgent/SpiceAgent.h

```diff
@@ -63,11 +64,13 @@ class SpiceAgent {
 
 private:
     NonnullOwnPtr<Core::File> m_spice_device;
-    ConnectionToClipboardServer& m_clipboard_connection;
     Vector<Capability> m_capabilities;
 
     RefPtr<Core::Notifier> m_notifier;
 
+    // Fired when we receive clipboard data from the spice server
+    ErrorOr<void> on_clipboard_message(ClipboardMessage& message);
```

Comment:
Generally the `on_foo` name pattern is used for `Function<void(blah)>` public callbacks that are meant to be hooked up by a user of the class or top level application code when wiring up the desired behavior. perhaps `forward_clipboard_message` ? `handle_clipboard_message` ?